assets/js/blocks/checkout/inner-blocks/checkout-fields-block/edit.tsx

```diff
@@ -41,16 +40,8 @@ export const Edit = ( {
 	const defaultTemplate = [
 		[ 'woocommerce/checkout-express-payment-block', {}, [] ],
 		[ 'woocommerce/checkout-contact-information-block', {}, [] ],
-		...[
-			isExperimentalBuild()
-				? [ 'woocommerce/checkout-shipping-method-block', {}, [] ]
-				: null,
-		],
-		...[
-			isExperimentalBuild()
-				? [ 'woocommerce/checkout-pickup-options-block', {}, [] ]
-				: null,
-		],
+		[ 'woocommerce/checkout-shipping-method-block', {}, [] ],
+		[ 'woocommerce/checkout-pickup-options-block', {}, [] ],
```

Comment:
We must do this so that once enabled, Pickup Location blocks are in the correct place.